python/ml4ir/tests/test_ranking_model.py

```diff
@@ -85,9 +94,9 @@ def test_csv_and_tfrecord(self):
         )
 
         # Check if the loss and accuracy on the test set is the same
-        assert np.isclose(tfrecord_loss, 0.55004, rtol=0.05)
-        assert np.isclose(tfrecord_mrr, 0.70067, rtol=0.05)
+        assert np.isclose(tfrecord_loss, 0.59263, rtol=0.01)
```

Comment:
oh look. it changed.  How am I supposed to review that this new value is "the right" one?  If someone else issues a PR, and this test fails because now the loss is 0.573, do they just update this test to say we now expect 0.573?  What are we even checking here?

Reply:
Haha. I don't know how else to check if the end to end pipeline works though. We will remove this once we have more unit tests added. Deal!

Reply:
I'm just not even sure these asserts are asserting anything of value.  If these three lines were replaced with `assert np.greaterthan(tfrecord_loss, 0.0)`, would you get the same information (i.e that the code made it to this point)?

Reply:
What I was hoping to check was... If someone set up `ml4ir` on their local box, this is a quick way to validate that the results are not too far off from what we have observed. I have fixed the data, the seeds, etc.

I can do a `np.greaterthan..` as well. But that would be a softer check.

Reply:
> If someone set up ml4ir on their local box, this is a quick way to validate that the results are not too far off from what we have observed. I have fixed the data, the seeds, etc.

Right, so how could this ever fail?  This "hard" check has to pass, right?  I guess it's a "point in time" test: any refactoring you do which leads to this test passing is basically guaranteed to have the same underlying ML under the hood, I guess?

But any change in which you *expect* the ML to change, even just slightly, will lead to this test failing, and the only fix would be to update the test with the new observed value, and "declare" that "this is the newly fixed Gold Standard", right? :\

Reply:
My assumption was that it would fail if the right dependencies(or the right versions) weren't set up. But you are right about the second part - any small change to the actual ML side of the code will result in this value changing. :/